scripts/lambda.txt

```diff
@@ -0,0 +1 @@
+[0,0.01,0.02,0.05,0.1]
```

Comment:
Let's make this one value per line as well so that the file formats are consistent